doc/source/user/numpy-for-matlab-users.rst

```diff
@@ -53,7 +53,7 @@ Some Key Differences
        things that Matlab just cannot, for instance dealing properly with
```

Comment:
```suggestion
       things that MATLAB just cannot, for instance dealing properly with
```
And throughout the file.

Reply:
Keep in mind there are also a couple of lowercase "matlab" instances that should be fixed.